src/index.html

```diff
@@ -1,12 +1,492 @@
+          </li>
+
+          <li class="nav__item">
+            <a
+             href="#tablets"
+             class="nav__link"
+            >
+             Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             href="#photo"
+             class="nav__link"
+            >
+             Photo
+            </a>
+          </li>
+          
+          <li class="nav__item">
+            <a
+             href="#video"
+             class="nav__link"
+            >
+             Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page__main">
+      <div class="card page__card" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 (MNED2UA/A)"
+        >
+
+        <h3 class="card__title">
+          <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        </h3>
+
+        <h3 class="card__subtitle">
+          <p>Product code: 195434</p>
+        </h3>
+
+        <div class="card__reting">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <h3 class="card__reviews">
+            <p>Reviews: 5</p>
+          </h3>
+        </div>
+
+        <div class="card__price">
+          <h3 class="card__price-title">
+            <p>Price:</p>
+          </h3>
+
+          <h3 class="card__price-cost">
+            <p>$2,199</p>
+          </h3>
+        </div>
+
+        <a
+          href="#"
+          class="card__link"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card page__card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 (MNED2UA/A)"
+        >
+
+        <h3 class="card__title">
+          <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        </h3>
+
+        <h3 class="card__subtitle">
+          <p>Product code: 195434</p>
+        </h3>
+
+        <div class="card__reting">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <h3 class="card__reviews">
+            <p>Reviews: 5</p>
+          </h3>
+        </div>
+
+        <div class="card__price">
+          <h3 class="card__price-title">
+            <p>Price:</p>
+          </h3>
+
+          <h3 class="card__price-cost">
+            <p>$2,199</p>
+          </h3>
+        </div>
+
+        <a
+          href="#"
+          class="card__link"
+          data-qa="hover"
```

Comment:
The 'data-qa' attribute should be 'data-qa="card-hover"' instead of 'data-qa="hover"'